tests/trestle/core/commands/validate_test.py

```diff
@@ -248,6 +251,15 @@ def test_validate_dup_uuids(sample_component_definition: ComponentDefinition) ->
         0].uuid
     assert not validator.model_is_valid(sample_component_definition, True)
 
+    assert validator.model_is_valid(sample_trestle_profile, True)
+
+    # add extra set_param with duplicate param_id and confirm it is not valid
+    # only profile has this additional test
+    set_param = sample_trestle_profile.modify.set_parameters[0].copy()
+    set_param.values = ['foo']
+    sample_trestle_profile.modify.set_parameters.append(set_param)
+    assert not validator.model_is_valid(sample_trestle_profile, True)
```

Comment:
May be add the test to assert that validation passes for the `sample_component_definition` from above if that cd has duplicate param_id. Just to solidify this logic for the future.